rust/otap-dataflow/crates/controller/src/lib.rs

```diff
@@ -321,11 +329,25 @@ impl<PData: 'static + Clone + Send + Sync + std::fmt::Debug> Controller<PData> {
         pipeline_ctrl_msg_tx: PipelineCtrlMsgSender<PData>,
         pipeline_ctrl_msg_rx: PipelineCtrlMsgReceiver<PData>,
     ) -> Result<Vec<()>, Error> {
+        // Create a tracing span for this pipeline thread
+        // so that all logs within this scope include pipeline context.
+        let span = otel_info_span!(
+            "pipeline_thread",
+            pipeline_group = %pipeline_key.pipeline_group_id,
+            pipeline = %pipeline_key.pipeline_id,
+            core_id = core_id.id,
+        );
```

Comment:
This `otel_info_span!` follows more or less the suggestion I made before. I will suggest to replace the `=` by `:` in order to be more aligned with the field syntax used for structs.